packages/cli/src/lib/snapshot/expandedPreviewer.ts

```diff
@@ -0,0 +1,85 @@
+import {
```

Comment:
in that case, I would rename `reportViewer` to `Previewer`. That way, we will have the `Previewer` and the `ExpandedPreviewer`

Reply:
Naming is really temporary, I'm mostly at the 'throw pasta on the wall and see what stick" stage.

And honestly, I don't think I would merge it till I can test it for real

Reply:
So @y-lakhdar, we go with `Previewer`?